lib/LaTeXML/Core/Gullet.pm

```diff
@@ -182,14 +182,15 @@ sub show_pushback {
 #**********************************************************************
 # Not really 100% sure how this is supposed to work
 # See TeX Ch 20, p216 regarding noexpand, \edef with token list registers, etc.
-# Solution: Duplicate param tokens, stick NOTEXPANDED infront of expandable tokens.
+# Solution: Duplicate param tokens, stick NOTEXPANDED infront of expandable+undefined tokens.
 sub neutralizeTokens {
   my ($self, @tokens) = @_;
   my @result = ();
   foreach my $token (@tokens) {
     if ($$token[1] == CC_PARAM) {    # Inline ->getCatcode!
       push(@result, $token); }
-    elsif (defined(my $defn = LaTeXML::Core::State::lookupDefinition($STATE, $token))) {
+    elsif (!defined(my $meaning = LaTeXML::Core::State::lookupMeaning($STATE, $token)) ||
```

Comment:
Isn't both meaning & defintion redundant?

Reply:
What I intended to add here, was also neutralizing undefined command sequences, which is what a negative `lookupMeaning` achieves.

If your point is that every time `lookupDefinition` is true, then `lookupMeaning` is also true, that may be.

The question is, are there cases where `lookupMeaning` is true, but `lookupDefinition` is false, where we do not want to add a `\noexpand`. An example of that is e.g. any `T_LETTER` token, and a variety of others. I think adding a `\noexpand` before those is simply a no-op, so it may be a "distinction without a difference"... Unsure what is cleanest.

Reply:
If we could get away with adding a `\noexpand` unconditionally, before every element of the input `@tokens`, that would certainly feel most elegant to me.

Reply:
Doh; missed the "!".